frontend/service.go

```diff
@@ -221,6 +220,24 @@ func (s *LwdStreamer) SendTransaction(ctx context.Context, rawtx *walletrpc.RawT
 	// Result:
 	// "hex"             (string) The transaction hash in hex
 
+	if common.DarksideIsEnabled() {
```

Comment:
It seems like this function should possibly be separate. Should we consider removing this `if common.DarksideIsEnabled()` check and creating a separate function, instead?